ferrocene/ci/configure.sh

```diff
@@ -193,11 +193,11 @@ if [[ is_internal ]]; then
     add --set target.aarch64-unknown-ferrocenecoretest.cc=aarch64-linux-gnu-gcc
     add --set target.aarch64-unknown-ferrocenecoretest.profiler=false # no profiling support
 
-    add --set target.thumbv7em-ferrocenecoretest-eabi.cc=arm-none-eabi-gcc
-    add --set target.thumbv7em-ferrocenecoretest-eabi.profiler=false # no profiling support
+    add --set target.thumbv7em.m4-ferrocenecoretest-eabi.cc=arm-none-eabi-gcc
```

Comment:
@pietroalbini Do you know how I can specify a target with a `.` in it without it becoming a big yak shave in `configure.py`?

Reply:
add --set target.\"thumbv7em.m4-ferrocenecoretest-eabi\".cc=arm-none-eabi-gcc